### PR TITLE
Fault and Twist controller from new repo

### PR DESCRIPTION
**Pull request by moriarty**
- We aim to have these two controllers merged into ros2_control
- For now get them from picknik_controllers repo

### CODE REVIEW

**Comment by moriarty** [comment]
Heads up @eholum Breaking this out into it's own thing might affect MoveIt Studio because of how it pulls in dependencies, it will need to have a dependency on picknik_controllers

File: README.md [hunk 1]
```diff
@@ -159,29 +159,11 @@ ros2 action send_goal /robotiq_gripper_controller/gripper_cmd control_msgs/actio
...
-### kortex_examples
-This package holds all the examples needed to understand the basics of `ros_kortex`. Most of the examples are written in both C++ and Python. Only the MoveIt! example is available exclusively in Python for now.
-A more detailed [description](kortex_examples/readme.md) can be found in the package subdirectory.
-
-### kortex_gazebo
-This package contains files to simulate the Kinova Gen3 and Gen3 lite robots in Gazebo. For more details, please consult the [README](kortex_gazebo/readme.md) from the package subdirectory.
-
-### kortex_move_it_config
+### kortex_moveit_config
 This metapackage contains the auto-generated MoveIt! files to use the Kinova Gen3 and Gen3 lite arms with the MoveIt! motion planning framework. For more details, please consult the [README](kortex_move_it_config/readme.md) from the package subdirectory.
```
Comment (abake48) [inline README.md]:
```suggestion
This metapackage contains the auto-generated MoveIt! files to use the Kinova Gen3 and Gen3 lite arms with the MoveIt! motion planning framework. For more details, please consult the [README](kortex_moveit_config/readme.md) from the package subdirectory.
```

**Review by abake48 — CHANGES_REQUESTED** [review]
one small change and then I'll approve

**Review by abake48 — APPROVED** [review]

**Comment by abake48** [comment]
Studio should be safe until we migrate over to the latest main. 

That will be a part of my todos for moving the Kinova robot configs out of the core Studio repo and into their own workspace repo

**Pull request merged**